File: cs50x/psets-labs/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
//Obtain height of pyramid b/w 1 and 8
    int h;
    do
    {
        h  = get_int("Height: ");
    }
//Prompt user to enter values only between 1 and 8
    while (h < 1 || h > 8);
//Build Pyramid
    for (int i = 0; i < h; i++)
    {
        for (int j = 0; j < h - i - 1; j++) //Leaves appropriate space
        {
            printf(" ");
        }

        for (int j = 0; j <= i; j++) // prints left pyramid
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)  //prints right pyramid
        {
            printf("#");
        }
        printf("\n");
    }

}

File: cs50x/psets-labs/sentimental-mario-more/mario.py
import cs50

while True:  # Get Height and ensure proper input
    h = cs50.get_int("Height: ")
    if h >= 1 and h <= 8:
        break

for i in range(1, h + 1):
    for j in range(h - i, 0, -1):  # Leave Neccessary space
        print(" ", end="")
    for j in range(1, i + 1):  # print left pyramid
        print("#", end="")
    print(" ", end=" ")  # leave appropriate gap
    for j in range(i, 0, -1):  # print right pyramid
        print("#", end="")
    print()
